### ISSUE TITLE
[Design] - Applicants, Managers, HR / Profile questions

### ISSUE THREAD

**Issue by nicolsonaitken** [GCTC-NTGC/TalentCloud#2366]
# Design Goal & User Groups
**Please explain the primary goals of this design work.**
It is unclear to both managers and HR advisors on what to do with the information provided within applicant profiles. In addition, applicants are also unclear as to what value this information brings and are simply leaving it empty in most cases. 

**Please select the users this UI will apply to.**
- [ ] Administrators
- [x] [Applicants](https://github.com/GCTC-NTGC/TalentCloud/issues/1548)
- [x] [Hiring Managers](https://github.com/GCTC-NTGC/TalentCloud/issues/1549)
- [x] [HR Advisers](https://github.com/GCTC-NTGC/TalentCloud/issues/1550)

# Initial Research & Notes
**Please list/link all research performed before this design work began. Include research relevant to insights from team members as well. Don't be afraid to ask for references or documents!**

-  

## Assumptions
**Please list all assumptions made during your design work. This should include behavioural assumptions made by the team that have no direct research references.**

- we assumed that...
- etc.

# Design Versions
Each version should contain the following:
- the version number
- a link to the prototype
- a list of all changes made since the last version and why those changes were made
- a screenshot of the new UI
- any research performed on this version, as well as outcomes for the next version

## Version 1

Prototype Link:

### Changes
**Please list all changes made to this version from the last, and indicate why that change was made when appropriate.**

- we added a bulleted list
  - this was done as a result of...
- etc.

### Research/Feedback & Outcomes
**Please include notes, links, images, etc. that are associated with any research performed on this version only. Outline any outcomes required in the next version.**

## Version 2
**Repeat versions as necessary.**

**Comment by nicolsonaitken** [comment]
Let's make sure there really is a good reason. May speak to need for broader redesign of profiles. Or maybe the applicants profile goes away — @gobyrne

**Comment by nicolsonaitken** [comment]
I went into my profile to go through the process again from the start and truth is, this is hard. I don't know what my learning journey is or more importantly, I don't know what's different in my learning journey versus my career journey. I didn't take any of this is in University. I just started a web company 20 years and went from there... The other questions are more clear but at the end of the day this feels like a) a lot of effort and b) a cover letter?
